clusters/app.ci/registry-access/ovirt/admin_ovirt_list.yaml

```diff
@@ -1,41 +0,0 @@
-kind: List
-apiVersion: v1
-items:
-- apiVersion: v1
-  kind: Namespace
-  metadata:
-    name: ovirt
-- apiVersion: authorization.openshift.io/v1
-  kind: RoleBinding
-  metadata:
-    name: image-puller
-    namespace: ovirt
-  roleRef:
-    name: system:image-puller
-  subjects:
-  - kind: SystemGroup
-    name: system:authenticated
-  - kind: SystemGroup
-    name: system:unauthenticated
```

Comment:
Do you want to migrate this rolebinding too? The things that you add allow the SA to pull images from `ocp` namespace, but you also had this binding that allowed public access to images in `ovirt`.

Reply:
Add it, thanks for the catch :)